Trappist/src/Promact.Trappist.Repository/Test/TestsRepository.cs

```diff
@@ -33,9 +43,67 @@ public TestsRepository(TrappistDbContext dbContext, IGlobalUtil util)
             return !isTestExists;
         }
 
+        /// <summary>
+        /// Fetch all the tests from Test Model,Convert it into List
+        /// </summary>
+        /// <returns>List of Testsby decreasing order of there created Date</returns>
         public async Task<List<Test>> GetAllTestsAsync()
         {
             return await _dbContext.Test.OrderByDescending(x => x.CreatedDateTime).ToListAsync();
         }
+
+        /// <summary>
+        /// Updates the edited Test Name
+        /// </summary>
+        /// <param name="id">The parameter "id" takes takes the value of the Id from the route</param>
+        /// <param name="testObject">The parameter "testObject" is an object of Test</param>
+        /// <returns>Updated Test Name</returns>
+        public async Task UpdateTestNameAsync(int id, Test testObject)
+        {
+            var testSettingsToUpdate = _dbContext.Test.FirstOrDefault(x => x.Id == id);
```

Comment:
Its being checked in the Controller that if the id does not exist then NotFound() status code will be returned.